src/main/java/Duke.java

```diff
@@ -1,10 +1,343 @@
+import java.io.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
+        File database = new File("./src/main/java/database.txt");
+
+        BufferedReader br = new BufferedReader(new FileReader(database));
+        ArrayList<Task> added = new ArrayList<Task>();
+
+        load(added, br);
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner sc = new Scanner(System.in);
+
+        while (true) {
+            String input = sc.nextLine();
+
+            try {
+                String firstWord = getFirstWord(input);
+            }
+            catch (DukeException e) {
+                System.out.println("Oops!! I'm sorry, but I don't know what that means :(");
+                continue;
+            }
+
+            if (input.equals("bye")) {
+                break;
+            }
+
+            else if (input.equals("list")) {
+                list(added);
+            }
```

Comment:
Try to use Egyptian style for brackets.
if (...) {
    //code
} else if (...) {
    //code
}